resources/ext/ezbake.conf

```diff
@@ -21,8 +21,9 @@ ezbake: {
                  "/opt/puppetlabs/puppet/bin/puppet config set --section master rundir  /var/run/puppetlabs/puppetserver",
                  "/opt/puppetlabs/puppet/bin/puppet config set --section master pidfile /var/run/puppetlabs/puppetserver/puppetserver.pid",
                  "/opt/puppetlabs/puppet/bin/puppet config set --section master codedir /etc/puppetlabs/code",
-                 "usermod --home /opt/puppetlabs/server/data/puppetserver puppet",
-                 "install --directory --owner=puppet --group=puppet --mode=775 /opt/puppetlabs/server/data"
+                 "install --directory /etc/puppetlabs/puppet/ssl",
+                 "chown -R puppet:puppet /etc/puppetlabs/puppet/ssl",
+                 "find /etc/puppetlabs/puppet/ssl -type d -print0 | xargs chmod 770"
```

Comment:
This is missing the `-0` between `xargs` and `chmod`, like so:
```sh
find  /etc/puppetlabs/puppet/ssl -type d -print0 | xargs -0 chmod 770
```

But...I'm wondering if this line is necessary at all? I tested these changes out on Cent 7 and things were working after the `chown`. I haven't tried other OSes yet though...

Here's what I did:
1. Install puppet-agent dc53fa36a1813454b9b55243a3f291dae122d614
2. puppet agent apply to create the /etc/puppetlabs/puppet/ssl directory (and its subdirectories)
3. Install puppet-server package from head of master
4. Start puppetserver service and see a Permission Denied exception when writing `ca.pem` to  `/etc/puppetlabs/puppet/ssl/certs`
5. `chown -R puppet:puppet /etc/puppetlabs/puppet/ssl`
6. Restart service successfully, do agent runs, etc.

Maybe I'm not accounting for some possibility though?